tests/unit/test_hnsw_tiered.cpp

```diff
@@ -2799,3 +2799,305 @@ TYPED_TEST(HNSWTieredIndexTestBasic, overwriteVectorAsync) {
         delete index_ctx;
     }
 }
+
+TYPED_TEST(HNSWTieredIndexTest, RangeSearch) {
+    size_t dim = 4;
+    size_t k = 11;
+    size_t per_label = TypeParam::isMulti() ? 5 : 1;
+
+    size_t n_labels = k * 3;
+    size_t n = n_labels * per_label;
+
+    auto edge_delta = (k - 0.5) * per_label;
+    auto mid_delta = edge_delta / 2.1;
```

Comment:
Why 2.1?

Reply:
an edge case in multi-value required some a bit higher value than 2